### DISCUSSION TITLE
Update for BlazorExample?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [MarimerLLC/csla#2743] [Questions]
I just did a Git clone and the BlazorExample references 6.0.0-R21120603. After logging in I get `Object reference not set to an instance of an object` on the **Add person** page. After updating the package references to R22012401 I get build errors in the client and server `Program.cs`.

Am I doing something wrong?

**Comment by rockfordlhotka** [comment]
Too many changes too fast to keep up - I need more help 😄 

Why it doesn't work against the old prerelease I don't know. I'm sure it won't work against the latest prerelease because we've changed a lot of things related to Blazor, especially in prereleases 3 and 4.
